### DISCUSSION TITLE
Get image urls with content API

### DISCUSSION THREAD

**Discussion by jymden** [statamic/cms#7139] [Q&A] [answered]
When returning an image asset from the REST API only the original url is included.  We have set up glide to generate a number of images from presets and need to find a way to get the url's for these images. 

The images are generated and put with hashed filenames under storage/statamic/glide/images/filename.jpg/

But when I get the asset from the api the only url included is for the original image. We can not use glide to serve the images with manipulation since we need to have secure mode on.

How do I actually get hold of the urls for the preset images?

**Comment by jasonvarga** [comment, marked as answer]
Customize the resource and add a field that has the url you need. https://statamic.dev/rest-api#customizing-resources
